task8/ServletsOnMaven/src/main/java/ServletsOnPosts/ServletsOnUsers.java

```diff
@@ -0,0 +1,39 @@
+package ServletsOnPosts;
+
+import Posts.UtilClasses.JSONDecorator;
+import Posts.UtilClasses.User;
+import WorkWithDB.WorkWithDB;
+import org.apache.commons.io.IOUtils;
+
+import javax.annotation.Resource;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+import java.io.IOException;
+
+public class ServletsOnUsers extends HttpServlet {
+
+    @Resource(name="jdbc/mydb")
+    private DataSource dataSource;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String username = req.getParameter("username");
+        if (username == null) {
+            resp.getWriter().write(JSONDecorator.toJson(null));
+            return;
+        }
+        String passwordUser = WorkWithDB.getPassword(username, dataSource);
```

Comment:
Can't still clearly understand what's going on here, but it worries me)
You either should name you variable in an explicit way (is this an entity describing *use*? then, it's `authenticatedUser` or such)
Or else, it looks like you are sending password over http. I hope you not